mix.exs

```diff
@@ -0,0 +1,78 @@
+defmodule App.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :app,
+      version: "0.1.0",
+      elixir: "~> 1.14",
+      elixirc_paths: elixirc_paths(Mix.env()),
+      start_permanent: Mix.env() == :prod,
+      aliases: aliases(),
+      deps: deps()
+    ]
+  end
+
+  # Configuration for the OTP application.
+  #
+  # Type `mix help compile.app` for more information.
+  def application do
+    [
+      mod: {App.Application, []},
+      extra_applications: [:logger, :runtime_tools]
+    ]
+  end
+
+  # Specifies which paths to compile per environment.
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_), do: ["lib"]
+
+  # Specifies your project dependencies.
+  #
+  # Type `mix help deps` for examples and options.
+  defp deps do
+    [
+      {:phoenix, "~> 1.7.0-rc.0", override: true},
+      {:phoenix_ecto, "~> 4.4"},
+      {:ecto_sql, "~> 3.6"},
+      {:postgrex, ">= 0.0.0"},
+      {:phoenix_html, "~> 3.0"},
+      {:phoenix_live_reload, "~> 1.2", only: :dev},
+      {:phoenix_live_view, "~> 0.18.3"},
+      {:heroicons, "~> 0.5"},
+      {:floki, ">= 0.30.0", only: :test},
+      {:phoenix_live_dashboard, "~> 0.7.2"},
+      {:esbuild, "~> 0.5", runtime: Mix.env() == :dev},
+      {:tailwind, "~> 0.1.8", runtime: Mix.env() == :dev},
+      {:swoosh, "~> 1.3"},
```

Comment:
@LuchoTurtle please get into the habit of _excluding_ the stuff we _don't_ need from `Phoenix` projects. 🙏 
e.g: 

```sh
mix phx.new app --no-dashboard --no-ecto --no-gettext --no-mailer 
```

All the recent updates include `swoosh` for sending emails.
That's one more thing for us to receive annoying @depedabot updates for and have to _maintain_.
Please be ultra-precise with what you include in a project/tutorial. 
This example isn't going to need a fancy `Phoenix` dashboard, translation or even DB so it can be _significantly_ streamlined. Which will then make deploying the example to Fly.io _far_ more efficient (no Postgres instance required). 💭

Reply:
You are completely right, I keep forgetting about this detail. Will do better next time.
We will be using a database, though. To keep track of the logged users who have paid or not paid.

Reply:
In a simple example I would be tempted to use **`ETS`** to store a JSON/Map of data: https://elixirschool.com/en/lessons/storage/ets
Running a separate VM for `Postgres` just to store **`2kb`** of data is like using a **Jet** to drive to the corner shop to buy bread. 😜

Reply:
ETS is in-memory. Unless you want to lose data when a server restarts, I could give it a whirl.

Reply:
Yeah, ETS is often in-memory only, but can easily be persisted to disk: https://elixirforum.com/t/persistentets-lightweight-persistence-over-ets/3951